### PR TITLE
Added the Explorer Path to the DPP-Verification tutorial for the Tractus-X Community Days

### PR DESCRIPTION
**Pull request by ELebedkin**
# Why we create this PR?
 
This PR is necessary to provide the latest version of the showcase demo for the Tractus-X Community Days taking place on December 5th and 6th.

# What we want to achieve with this PR?

The goal of this PR is to offer a suitable, non-technical approach that allows business users, experts, and innovation enthusiasts to easily try out the DPP verification process.
 
# What is new?
 
- Introduced a new tutorial path called "Explorer Path" (explorer-path-insomnia).
- Added a data payload link for downloading the Insomnia API calls.
- Improved the overall process with a smoother and more non-technical explanation of the steps involved.

## PR Linked to:

-

### CODE REVIEW

**Comment by matbmoser** [comment]
Please pull the changes from main, I updated our dependency files and workflows

**Review by matbmoser — CHANGES_REQUESTED** [review]
For every image we need to add a .license. In general looks really good, we could merge once it's there. Please make sure that the files do not contain spaces, this can bring problems for different OS.

**Comment by ELebedkin** [comment]
@matbmoser I have pulled the changes, added to every image in the tutorial an additional license file - as you have mentioned it - and improved the whole process (explorer + builders path) and completed it. so there is no separate aspect-model.md or digital-twin.md, they are now a part of the explorer and builders path, which are at the end linking back to the readme.md file step 3 generating a qr-code. 

Due to the confussion with the many commits: i swapped test and main branch. 
It should now be up2date.

File: dpp-tutorial/resources/screenshots/generate_licenses_python_script/generate_licenses.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    if filename.endswith((".png", ".jpg", ".jpeg")):
+        # Construct the path for the .license file
+        license_file = os.path.join(image_folder, f"{filename}.license")
+        
+        # Write the license template to the .license file
+        with open(license_file, "w") as file:
+            file.write(license_template)
+        print(f"Generated new license file: {license_file}")
+
+print("All old license files deleted and new license files generated!")
```
Comment (matbmoser) [inline dpp-tutorial/resources/screenshots/generate_licenses_python_script/generate_licenses.py:35]: Please do not commit this!

Also use the following license template:

```
## NOTICE

This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).

- SPDX-License-Identifier: CC-BY-4.0
- SPDX-FileCopyrightText: 2023, 2024 BMW AG
- SPDX-FileCopyrightText: 2023, 2024 CGI Deutschland B.V. & Co. KG
- SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
- Source URL: https://github.com/eclipse-tractusx/digital-product-pass

```

File: dpp-tutorial/resources/screenshots/agreebutton.png.license [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Author: Mathias Brunkow Moser
+Description: This image was created to support the dpp-tutorial.
+License:
+- SPDX-License-Identifier: CC-BY-4.0
+- SPDX-FileCopyrightText: 2024 CGI Deutschland B.V. & Co. KG
+- Source URL: https://github.com/eclipse-tractusx/digital-product-pass
```
Comment (matbmoser) [inline dpp-tutorial/resources/screenshots/agreebutton.png.license:6]: Please use this template:

```md
## NOTICE

This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).

- SPDX-License-Identifier: CC-BY-4.0
- SPDX-FileCopyrightText: 2023, 2024 BMW AG
- SPDX-FileCopyrightText: 2023, 2024 CGI Deutschland B.V. & Co. KG
- SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
- Source URL: https://github.com/eclipse-tractusx/digital-product-pass

```

**Review by matbmoser — CHANGES_REQUESTED** [review]

**Review by matbmoser — CHANGES_REQUESTED** [review]
Try to pack everything in some commits 88 is much for the fewer changes you made

File: dpp-tutorial/explorer-path.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+SPDX-License-Identifier: CC-BY-4.0
+
+#######################################################################
+-->
+
+# Explorer Path
+
+In this Path, you, as a provider, will create a Digital Product Passport (DPP) for a specific Part of the Arena-X Car. For this purpose, you will use <a href="https://insomnia.rest/" target="_blank">Insomnia</a>, a user-friendly tool for sending and receiving HTTP requests.
+
+> [!Tip]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:27]:
```suggestion
> [!TIP]
```

File: dpp-tutorial/explorer-path.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+      "Placed on Market Date": "15.01.2024",
+      "List of Materials": "Aluminum",
+      "Hazard Materials": "Lead, Butyl, Cyanoacrylates, Polyurethane",
+      "Guarantee": "24 months"
+  }
+}
+
+```
+
+> [!Note]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:86]:
```suggestion
> [!NOTE]
```

File: dpp-tutorial/explorer-path.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+
+### Data Mapping Table
+
+Replace now every placehlders given in the data mapping template with the data from your part. 
+Use the following table to identify where to place your part's information in the template:
+
+> [!TIP]
+> You can search for the placeholder `CTRL + F` on Windows or `CMD + F` on MacOS/Linux
+
+> [!Note]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:122]:
```suggestion
> [!NOTE]
```

File: dpp-tutorial/explorer-path.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+- Digital Twin Registry (DTR):
+
+The Digital Twin Registry is required to manage and store Digital Twins (DT) in the form of Asset Administration Shells (AAS). This component ensures that the Digital Twins are properly registered and accessible for interaction within the ecosystem. The DTR is fully operational and preconfigured to handle incoming Digital Twin data.
+
+- Data Service (DS) (Data Provider):
+
+The Data Service is a crucial component for storing the payloads of Digital Product Passports in plain JSON format. This service should already be running and ready to accept data submissions. It acts as the backend repository where all passport-related information is securely stored and easily retrievable.
+
+
+> [!Caution]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:169]:
```suggestion
> [!CAUTION]
```

File: dpp-tutorial/explorer-path.md [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+
+---
+
+### Step 2.2 
+
+Now we actually will create the digitil Twin.
+
+1. Open the Insomnia request labeled `Step 2.2.1 Create Digital Twin"`.
+
+> [!Note]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:208]:
```suggestion
> [!NOTE]
```

File: dpp-tutorial/explorer-path.md [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+
+```bash
+<PART_INSTANCE_ID>                     ->   the value of part instance written on datasheet
+<PART_NAME>                            ->   the part number is written on the datasheet from a part
+<digitalTwinId>                        ->   the UUID written on datasheet
+<digitalTwinSubmodelId>                ->   the UUID generated the step ago
+```
+
+
+> [!Important]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:223]:
```suggestion
> [!IMPORTANT]
```

File: dpp-tutorial/explorer-path.md [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+> There are **two instances** of `digitalTwinSubmodelId` in the example. Please replace **both** of them:
+> - One is used as `"id"`
+> - The other is used as `"href"`
+
+> [!Important]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:228]:
```suggestion
> [!IMPORTANT]
```

File: dpp-tutorial/explorer-path.md [hunk 10]
```diff
@@ -0,0 +1,292 @@
...
+> There are **two instances** of `digitalTwinId` in the example. Please replace **both** of them:
+> - One is used as `"id"`
+> - The other is used as `"globalAssetId"`
+
+4. Send the POST request to add the Digital Twin to the Digital Twin Registry (DTR).
+- A successful request will return a `201 Created` response, which confirms that the Digitil Twin has been created successfully.
+
+> [!Note]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:236]:
```suggestion
> [!NOTE]  
```

File: dpp-tutorial/explorer-path.md [hunk 11]
```diff
@@ -0,0 +1,292 @@
...
+
+4. Send the request. A `200 OK` response confirms that the Digital Twin has been successfully registered.
+
+---
+
+### Step 2.3
+
+If you encounter an error or need to update the Digital Twin, you can use the Insomnia request labeled `Step 2.3 Modify Digital Twin` to make changes.
+
+> [!Note]
```
Comment (matbmoser) [inline dpp-tutorial/explorer-path.md:268]:
```suggestion
> [!NOTE]  
```

File: dpp-tutorial/builder-path.md [hunk 12]
```diff
@@ -0,0 +1,359 @@
...
+
+## Clone a Git repository
+
+Use the following command in your terminal to clone the digital product pass git repository
+
+```bash
+git clone https://github.com/eclipse-tractusx/digital-product-pass.git
+```
+
+> [!Note]
```
Comment (matbmoser) [inline dpp-tutorial/builder-path.md:56]:
```suggestion
> [!NOTE]  
```

File: dpp-tutorial/builder-path.md [hunk 13]
```diff
@@ -0,0 +1,359 @@
...
+
+## 1º Prepare Digital Product Pass Model
+
+In order to prepare aspect models, please follow the [aspect model guide](./aspect-model.md) 
+
+## Add Aspect Model to the Submodel Server
+
+The data generated in previous step can be stored into the submode data service.
+
+> [!Caution]
```
Comment (matbmoser) [inline dpp-tutorial/builder-path.md:166]:
```suggestion
> [!CAUTION]
```

File: dpp-tutorial/builder-path.md [hunk 14]
```diff
@@ -0,0 +1,359 @@
...
+> The UUID should be written in the format: 6fb9a71b-aee6-4063-a82e-957022aeaa7a
+
+> [!Note]
```
Comment (matbmoser) [inline dpp-tutorial/builder-path.md:169]:
```suggestion
> [!NOTE]
```

**Review by matbmoser — CHANGES_REQUESTED** [review]
Please change this to uppercase :)
But wonderful job! 🚀

**Review by matbmoser — APPROVED** [review]
LGTM, let merge this! Wonderful job @ELebedkin

**Pull request merged**